### DISCUSSION TITLE
第6节课作业(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#467] [作业]
第6节课作业(24班)

**Comment by waurblove** [comment]
https://github.com/waurblove/LLM/blob/main/day6.md

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/8ac545cd-c6a4-4dd8-acba-9a0d9c3bdd58

**Comment by yansuoyuli** [comment]
https://blog.csdn.net/m0_58431543/article/details/135761162

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135774472?spm=1001.2014.3001.5501

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day6_0.md

**Comment by flh1107** [comment]
https://www.bilibili.com/read/cv30012827/

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135845517?spm=1001.2014.3001.5501

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135846476?spm=1001.2014.3001.5501

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/679798588

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135850986
